docs/index.md

```diff
@@ -412,6 +412,9 @@ a different working order is required.
   This will also cut the diagram definition by half.
   See for example "`The book of bobbin lace stitches`" by Bridget M. Cook and Geraldine Stott. 
 
+### H16 variations
```

Comment:
Whoops, forgot the dash.  So far these sections are in numerical order, it might be better to have shortest first.

Reply:
```suggestion
### <a name="h18-variants"> H16  &mdash; variations
```

Reply:
H16 or H18?